lineage.yml

```diff
@@ -0,0 +1,7 @@
+Churn Model API Endpoint:
```

Comment:
We have never tracked this file for this AMP before -- I believe this is generated during the AMP setup process? Unless there is a good reason for including it now, I think this lineage.yml file should be removed from this PR